.github/workflows/ci.yml

```diff
@@ -51,10 +51,29 @@ jobs:
       run: |
         tox -e py
 
+  macos-versions:
+    name: macOS compatibility test
+    needs: smoke
+    runs-on: macos-11.0
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.5
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
```

Comment:
There's a discrepancy between the commend on line 60, and the version actually in use here; is there a particular reason you've used Python 3.7 for the Big Sur tests?

Reply:
Oops, I forgot to update the version in the `name`. (It would be really nice if GH Actions could auto-generate useful names for actions based on their settings so this wouldn't happen...)

Python 3.7 is the earliest available version on macOS 11 on GitHub Actions - see d37e35d82c9322564242cf4ff63ff69e0dc8bc5b.